### PR TITLE
Feature/rw random notification

### LINKED ISSUE #93: Add random notifications setting option
### Feature Categories:
- React

**Describe the feature/Bug your gonna work on**
- Create function that can grab a random notification audio sound
- Add additional option for "Random sound"

### PR DESCRIPTION
**Pull request by rwilliams659**
# Som Timer Pull Request Form

## Description
- Functionality added so a user can select the option to receive a random audio notification sound when the timer is up

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`
## Has This Been Tested?
- [ ] No
- [X] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:     
- Tested manually - both to ensure random sound plays in settings & when timer is up, in both localhost 3000 & 5000 environments 
# Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

resolve #93

### CODE REVIEW

File: src/Components/Settings/Settings.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ const Settings = () => {
           <option value={'pianoDreams'}>Piano Dreams</option>
           <option value={'levelUp'}>Level Up</option>
           <option value={'birdChord'}>Bird Chord</option>
+          <option value={'random'}>Randomize My Sounds</option>
```
Comment (Abdeboskey) [inline src/Components/Settings/Settings.js:76]: Ooooh very nice! I was imagining a button or checkbox or something but I like this much better.

File: src/helpers/audioHelper.js [hunk 2]
```diff
@@ -26,10 +33,11 @@ const playAlertSound = (sound) => {
     case "birdChord":
       audioNotification = new Audio(birdChord);
       break;
+    case "random":
+      audioNotification = new Audio(getRandomNotification(notificationSounds))
+      break;
```
Comment (Abdeboskey) [inline src/helpers/audioHelper.js:38]: Nicely done Rachel, really elegant and minimal solution 🙌🏼

File: src/apiCalls.js [hunk 3]
```diff
@@ -18,7 +18,6 @@ export const getRandomContent = (duration, category) => {
 }
 
 export const postSession = (session) => {
-  console.log(session)
```
Comment (Abdeboskey) [inline src/apiCalls.js:21]: Oooh yeah get that lil sucker outta there.

**Review by Abdeboskey — APPROVED** [review]
So speedy, so efficient, so nice! Thank you for doing this!

File: src/Components/Settings/Settings.js (hunk 1 above)
Comment (sienna-kopf) [reply src/Components/Settings/Settings.js:76]: OOh i like this too! Having it in the dropdown is cool

File: src/helpers/audioHelper.js [hunk 4]
```diff
@@ -5,7 +5,14 @@ import reverbSplash from "../assets/audio/reverbSplash.mp3";
 import levelUp from "../assets/audio/levelUp.mp3";
 import birdChord from "../assets/audio/birdChord.mp3";
 
-const playAlertSound = (sound) => {
+const notificationSounds = [balineseGong, goodOldSynths, pianoDreams, reverbSplash, levelUp, birdChord]
+
+const getRandomNotification = (sounds) => {
+  const randomIndex =  Math.floor(Math.random() * sounds.length)
```
Comment (sienna-kopf) [inline src/helpers/audioHelper.js:11]: Math.random() yay! Love when this stuff comes kinda built in.

**Review by sienna-kopf — APPROVED** [review]
Nice work @rwilliams659 !! This is really clean, clear, and such a cool simple feature!

**Review by sciencefixion — APPROVED** [review]

**Review by jkwest-93 — APPROVED** [review]
Great work!! Thanks for getting this feature figured out.

**Pull request merged**